lib/pure/mimetypes.nim

```diff
@@ -512,6 +512,14 @@ proc getExt*(mimedb: MimeDB, mimetype: string, default = "txt"): string =
     if m == mimetype:
       result = e
 
+proc getMediaType*(mimedb: MimeDB, ext: string, default = "text"): string =
```

Comment:
```
type MediaTypes = enum 
    application, audio, chemical, image, message, model, text, video ## these are valid 
```
 But the items with "-" (like `x-conference`) how should them beeing transcripted?
    xConference, xEpoc, xEorld?

And if the MimeDb gets filled with custom mimetypes? Can the enum mabe be created on compile time?